diff -ur moin-1.5.0beta4.orig/MoinMoin/PageEditor.py moin-1.5.0beta4/MoinMoin/PageEditor.py
--- moin-1.5.0beta4.orig/MoinMoin/PageEditor.py	2005-11-25 01:27:14.612927500 +0100
+++ moin-1.5.0beta4/MoinMoin/PageEditor.py	2005-11-25 01:34:05.810625750 +0100
@@ -402,14 +402,9 @@
         self.request.write("</form>")
         
         # QuickHelp originally by Georg Mischler <schorsch@lightingwiki.com>
-        self.request.write('<div id="editor-help">\n' + _(""" Emphasis:: [[Verbatim('')]]''italics''[[Verbatim('')]]; [[Verbatim(''')]]'''bold'''[[Verbatim(''')]]; [[Verbatim(''''')]]'''''bold italics'''''[[Verbatim(''''')]]; [[Verbatim('')]]''mixed ''[[Verbatim(''')]]'''''bold'''[[Verbatim(''')]] and italics''[[Verbatim('')]]; [[Verbatim(----)]] horizontal rule.
- Headings:: [[Verbatim(=)]] Title 1 [[Verbatim(=)]]; [[Verbatim(==)]] Title 2 [[Verbatim(==)]]; [[Verbatim(===)]] Title 3 [[Verbatim(===)]];   [[Verbatim(====)]] Title 4 [[Verbatim(====)]]; [[Verbatim(=====)]] Title 5 [[Verbatim(=====)]].
- Lists:: space and one of: * bullets; 1., a., A., i., I. numbered items; 1.#n start numbering at n; space alone indents.
- Links:: [[Verbatim(JoinCapitalizedWords)]]; [[Verbatim(["brackets and double quotes"])]]; url; [url]; [url label].
- Tables:: || cell text |||| cell text spanning 2 columns ||;    no trailing white space allowed after tables or titles.
-
-(!) For more help, see HelpOnEditing or SyntaxReference.
-""") + '\n</div>\n')
+        self.request.write('<div id="editor-help">\n')
+        Page (self.request, "QuickHelp").send_page (self.request, content_only=1)
+        self.request.write('\n</div>\n')
 
         if preview is not None:
             if staytop:
