diff -urN moin-1.5.4/wiki/config/wikiconfig.py moin-1.5.4.new/wiki/config/wikiconfig.py
--- moin-1.5.4/wiki/config/wikiconfig.py	2006-06-25 15:27:26.000000000 +0200
+++ moin-1.5.4.new/wiki/config/wikiconfig.py	2006-07-10 23:53:23.245458500 +0200
@@ -65,14 +65,14 @@
 
     # Where your mutable wiki pages are. You want to make regular
     # backups of this directory.
-    data_dir = './data/'
+    data_dir = '/var/lib/moin/data/'
 
     # Where read-only system and help page are. You might want to share
     # this directory between several wikis. When you update MoinMoin,
     # you can safely replace the underlay directory with a new one. This
     # directory is part of MoinMoin distribution, you don't have to
     # backup it.
-    data_underlay_dir = './underlay/'
+    data_underlay_dir = '/var/lib/moin/underlay/'
 
     # Location of your STATIC files (css/png/js/...) - you must NOT use the
     # same for invoking moin.cgi (or, in general, the moin code).
diff -urN moin-1.5.4/wiki/htdocs/index.html moin-1.5.4.new/wiki/htdocs/index.html
--- moin-1.5.4/wiki/htdocs/index.html	2006-05-11 18:24:01.000000000 +0200
+++ moin-1.5.4.new/wiki/htdocs/index.html	2006-07-10 23:52:36.442533500 +0200
@@ -1,11 +1,11 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
 <html>
   <head>
-    <meta http-equiv="refresh" content="0;URL=moin.cgi/">
+    <meta http-equiv="refresh" content="0;URL=./">
   </head>
 
   <body bgcolor="#FFFFFF" text="#000000">
-    Click <a href="moin.cgi">here</a> to get to the FrontPage.
+    Click <a href="./">here</a> to get to the FrontPage.
   </body>
 </html>
 
diff -urN moin-1.5.4/wiki/server/moin moin-1.5.4.new/wiki/server/moin
--- moin-1.5.4/wiki/server/moin	2006-06-25 18:19:54.000000000 +0200
+++ moin-1.5.4.new/wiki/server/moin	2006-07-10 23:46:23.495225750 +0200
@@ -16,6 +16,7 @@
 
 # Add the path to moin.py directory 
 ## sys.path.insert(0, '/path/to/moin')
+sys.path.insert(0, '/usr/share/moin/server')
 
 from MoinMoin.server import daemon
 from moin import Config, run
diff -urN moin-1.5.4/wiki/server/moin.cgi moin-1.5.4.new/wiki/server/moin.cgi
--- moin-1.5.4/wiki/server/moin.cgi	2006-06-25 15:31:27.000000000 +0200
+++ moin-1.5.4.new/wiki/server/moin.cgi	2006-07-10 23:47:26.083137250 +0200
@@ -17,7 +17,7 @@
 
 # Path of the directory where wikiconfig.py is located.
 # YOU NEED TO CHANGE THIS TO MATCH YOUR SETUP.
-sys.path.insert(0, '/path/to/wikiconfig')
+sys.path.insert(0, '/etc/webapps/moinmoin')
 
 # Path of the directory where farmconfig.py is located (if different).
 ## sys.path.insert(0, '/path/to/farmconfig')
diff -urN moin-1.5.4/wiki/server/moin.fcg moin-1.5.4.new/wiki/server/moin.fcg
--- moin-1.5.4/wiki/server/moin.fcg	2006-06-25 15:31:52.000000000 +0200
+++ moin-1.5.4.new/wiki/server/moin.fcg	2006-07-10 23:47:46.328402500 +0200
@@ -17,7 +17,7 @@
 
 # Path of the directory where wikiconfig.py is located.
 # YOU NEED TO CHANGE THIS TO MATCH YOUR SETUP.
-sys.path.insert(0, '/path/to/wikiconfig')
+sys.path.insert(0, '/etc/webapps/moinmoin')
 
 # Path of the directory where farmconfig is located (if different).
 ## sys.path.insert(0, '/path/to/farmconfig')
diff -urN moin-1.5.4/wiki/server/moinmodpy.py moin-1.5.4.new/wiki/server/moinmodpy.py
--- moin-1.5.4/wiki/server/moinmodpy.py	2006-06-25 15:32:57.000000000 +0200
+++ moin-1.5.4.new/wiki/server/moinmodpy.py	2006-07-10 23:49:12.673798750 +0200
@@ -37,7 +37,7 @@
 
 # Path of the directory where wikiconfig.py is located.
 # YOU NEED TO CHANGE THIS TO MATCH YOUR SETUP.
-sys.path.insert(0, '/path/to/wikiconfig')
+sys.path.insert(0, '/etc/webapps/moinmoin')
 
 # Path of the directory where farmconfig is located (if different).
 ## sys.path.insert(0, '/path/to/farmconfig')
diff -urN moin-1.5.4/wiki/server/moin.py moin-1.5.4.new/wiki/server/moin.py
--- moin-1.5.4/wiki/server/moin.py	2006-06-25 15:32:14.000000000 +0200
+++ moin-1.5.4.new/wiki/server/moin.py	2006-07-10 23:48:03.413470250 +0200
@@ -16,7 +16,7 @@
 
 # Path of the directory where wikiconfig.py is located.
 # YOU NEED TO CHANGE THIS TO MATCH YOUR SETUP.
-sys.path.insert(0, '/path/to/wikiconfig')
+sys.path.insert(0, '/etc/webapps/moinmoin')
 
 # Path of the directory where farmconfig is located (if different).
 ## sys.path.insert(0, '/path/to/farmconfig')
diff -urN moin-1.5.4/wiki/server/mointwisted moin-1.5.4.new/wiki/server/mointwisted
--- moin-1.5.4/wiki/server/mointwisted	2006-06-25 18:20:44.000000000 +0200
+++ moin-1.5.4.new/wiki/server/mointwisted	2006-07-10 23:50:03.420970250 +0200
@@ -13,6 +13,7 @@
 
 # Add the path to mointwisted.py directory 
 ## sys.path.insert(0, '/path/to/mointwisted')
+sys.path.insert(0, '/usr/share/moin/server')
 
 import os
 
diff -urN moin-1.5.4/wiki/server/mointwisted.py moin-1.5.4.new/wiki/server/mointwisted.py
--- moin-1.5.4/wiki/server/mointwisted.py	2006-06-25 15:34:03.000000000 +0200
+++ moin-1.5.4.new/wiki/server/mointwisted.py	2006-07-10 23:51:27.086199000 +0200
@@ -18,7 +18,7 @@
 
 # Path of the directory where wikiconfig.py is located.
 # YOU NEED TO CHANGE THIS TO MATCH YOUR SETUP.
-sys.path.insert(0, '/path/to/wikiconfig')
+sys.path.insert(0, '/etc/webapps/moinmoin')
 
 # Path to the directory where farmconfig is located (if different).
 ## sys.path.insert(0, '/path/to/farmconfig')
